ga4gh/backend.py

```diff
@@ -139,12 +168,9 @@ def _getContainer(self):
             raise exceptions.ReadGroupNotFoundException(readGroupId)
         return readGroup
 
-    def _getIterator(self):
-        iterator = self._container.getReadAlignments(
-            self._request.referenceName,
-            self._request.referenceId,
-            self._startPosition, self._request.end)
-        return iterator
+    def _search(self, start, end):
+        return self._container.getReadAlignments(
```

Comment:
Doh! _That's_ why the cherry pick didn't apply cleanly. Great catch @dcolligan, thanks for spotting that one. Should be fixed now.